scripts/romanisim-make-stack

```diff
@@ -0,0 +1,267 @@
+
+                        # add line to script
+                        spcript_file.write(line+'\n')
+
+                    else:
+                        # Obtain WCS catalog
+                        if args.catalog is None:
+                            twcs = wcs.get_wcs(metadata, usecrds=args.usecrds)
+                            rd_sca = twcs.toWorld(galsim.PositionD(
+                                galsim.roman.n_pix / 2, galsim.roman.n_pix / 2))
+                            cat = catalog.make_dummy_table_catalog(
+                                rd_sca, bandpasses=[entry['optical_element']], nobj=args.nobj)
+                        else:
+                            log.warning('Catalog input will probably not work unless the catalog '
+                                        'covers a lot of area or you have thought carefully about '
+                                        'the relation between the boresight and the SCA locations.')
+                            cat = Table.read(args.catalog)
+
+                        # If selected, apply persistence
+                        if args.previous:
+                            # Use provided Persistence file for first SCA (for each entry) only
+                            # after that, use previously created file
+                            if args.sca > 0 and sca > 1:
+                                persist = persistence.Persistence.read(previous_file_name)
+                            else:
+                                persist = persistence.Persistence.read(args.previous)
+
+                            previous_file_name = output_file_name
+                        else:
+                            persist = persistence.Persistence()
+
+                        rng = galsim.UniformDeviate(args.rng_seed)
+
+                        im, simcatobj = image.simulate(
+                            metadata, cat, usecrds=args.usecrds,
+                            webbpsf=args.webbpsf, level=args.level, persistence=persist,
+                            rng=rng)
+
+                        # Populate romanisim parameters and simulated object catalog
+                        romanisimdict = vars(args)
+                        romanisimdict['simcatobj'] = simcatobj
+
+                        # Write output asdf file with simulated data
+                        af = asdf.AsdfFile()
+                        af.tree = {'roman': im, 'romanisim': romanisimdict}
+                        af.write_to(output_file_name)
+
+
+                    # Break the loop if only one detector specified
+                    if args.sca > 0:
+                        break
+                    else:
+                        sca += 1
+
+                # Calculate time offset for the next exposure
+                openshuttertime = parameters.read_time * (
+                        ma_table[-1][0] + ma_table[-1][1] - 1)
+                time_offset += (openshuttertime + (float(entry['overhead_time']))) * u.s
```

Comment:
Let's use the 'duration' column I proposed here:
https://jira.stsci.edu/browse/APT-93485
So this would just be time_offset += entry['duration'] * u.s, I think.

Reply:
Implemented.